config/install/editor.editor.full_html.yml

```diff
@@ -1,10 +1,13 @@
+uuid: c8b180c0-46de-40dd-b717-1b0058af40ff
 langcode: en
 status: true
 dependencies:
   config:
     - filter.format.full_html
   module:
     - ckeditor
+_core:
+  default_config_hash: weN5_bB7CIiMYvkrDr44V2_YdjKeBG4yHcFuo9zoqKQ
```

Comment:
We need to manually remove the _core default_config_hash lines because this will be used in a distribution instead of individual site.

Reply:
This has been fixed now